manifests/kustomize/README.md

```diff
@@ -77,17 +60,62 @@ data:
             ...
 ```
 
-Or you can add it in one line with:
+Now you should be able to see the Model Registry UI in the Kubeflow Central Dashboard, and access to the Model Registry deployment in the profile namespace.
+
+### Uninstall
+
+To uninstall the Kubeflow Model Registry run:
 
 ```bash
-kubectl get configmap centraldashboard-config -n kubeflow -o json | jq '.data.links |= (fromjson | .menuLinks += [{"icon": "assignment", "link": "/model-registry/", "text": "Model Registry", "type": "item"}] | tojson)' | kubectl apply -f - -n kubeflow
-````
+# Uninstall Model Registry Instance
+PROFILE_NAME=<your-profile>
+for DIR in options/istio overlays/db ; do (cd $DIR; kustomize edit set namespace $PROFILE_NAME; kubectl delete -k .); done
+
+# Uninstall Model Registry UI
+kubectl delete -k options/ui/overlays/istio -n kubeflow
+```
+
+
+## Model Registry as a separate component Installation
+
+The following instructions will summarize how to deploy Model Registry as separate component in the context of a default Kubeflow >=1.9 installation.
+
+```bash
+kubectl apply -k overlays/db
+```
+
+As the default Kubeflow installation provides an Istio mesh, apply the necessary manifests:
+
+```bash
+kubectl apply -k options/istio
+```
+
+Check everything is up and running:
+
+```bash
+kubectl wait --for=condition=available -n kubeflow deployment/model-registry-deployment --timeout=2m
+kubectl logs -n kubeflow deployment/model-registry-deployment
+```
+
+Optionally, you can also port-forward the REST API container port of Model Registry to interact with it from your terminal:
+
+```bash
+kubectl port-forward svc/model-registry-service -n kubeflow 8081:8080
+```
+
+And then, from another terminal:
+
+```bash
+curl -sX 'GET' \
+  'http://localhost:8081/api/model_registry/v1alpha3/registered_models?pageSize=100&orderBy=ID&sortOrder=DESC' \
+  -H 'accept: application/json' | jq
+```
 
-## Usage
+### Usage
 
 For a basic usage of the Kubeflow Model Registry, follow the [Kubeflow Model Registry getting started documentation](https://www.kubeflow.org/docs/components/model-registry/getting-started/)
 
-## Uninstall
+### Uninstall
```

Comment:
shouldn't all the uninstalls be one place? can we please collate them in single section?

Reply:
These uninstalls are for the two sections, if we wanna remove the latter one this won't be needed, they are different because there's different steps.